gateway/src/connectors/defira/defira.ts

```diff
@@ -13,37 +13,21 @@ import {
   ContractTransaction,
 } from '@ethersproject/contracts';
 import {
-  // TODO: use this instead of UniswapRouter
-  Router,
-  // TODO: use this instead of UniswapPair. Note that its constructor takes in an optional factory address
+  // TODO: use this instead of DefiraRouter
+  Router as DefiraRouter,
+  // TODO: use this instead of DefiraPair. Note that its constructor takes in an optional factory address
   // parameter that SHOULD be populated since the default only works for mainnet
-  Pair,
+  Pair as DefiraPair,
   SwapParameters,
-  // TODO: use this instead of UniswapTrade
-  Trade,
+  // TODO: use this instead of DefiraTrade
+  Trade as DefiraTrade,
```

Comment:
Not consistent with how other connectors do it... but I kind of like this better.